lib/src/infinite_list.dart

```diff
@@ -76,6 +77,12 @@ class InfiniteList extends StatelessWidget {
   /// {@endtemplate}
   final bool reverse;
 
+  /// {@template shrinkWrap}
+  /// Indicates if the extent of the [ScrollView] in the [scrollDirection]
+  /// should be determined by the contents being viewed.
+  /// {@endtemplate}
```

Comment:
Nit: consider referencing the property that the value is proxying to.

```suggestion
  /// should be determined by the contents being viewed.
  ///
  /// See also:
  /// 
  /// * [CustomScrollView.shrinkWrap], for more details about this flag.
```

Reply:
Are you waiting for my response?

I not sure how to "add some tests to ensure the property gets proxied into CustomScrollView and ensure false is the default?"

Reply:
Hi @limcheekin sorry for the late reply, you can try having a look at a recent PR: https://github.com/VeryGoodOpenSource/very_good_infinite_list/pull/76, to get inspired on how to test it.